From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@alex-frankel__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

AjayKumar-N committed:
(ForumsTeam) A note to highlight if the procedure applies to Managed or UnManaged disks

The procedure (5.2 Get the shared access signature URI for your VM images) works fine for Unmanaged disks only, a note stating the same would be beneficial to the customers. 
Or highlight an alternate procedure for the Managed disks if available. 

If they choose Managed disks, the side effect of doing so is that there is no storage account to share during the publishing process.

There has been a similar issue raised by a customer on the forum here: https://social.msdn.microsoft.com/Forums/en-US/946e8bff-2d06-4b0f-b71e-cc833c7b3c3c/cant-see-the-offer-vhd-anymore?forum=WAVirtualMachinesforWindows
And, another customer 'commented' in the document about the same note.

Ajay, ForumsTeam

dbreshears committed:
Updating the Docker Tools Troubleshooting Doc

dbreshears committed:
Fixing links

dbreshears committed:
Formatting revisions

dbreshears committed:
moving where media located and more formatting issues

dbreshears committed:
Added note for resetting credentials

dbreshears committed:
Addressing Steven's comments

dbreshears committed:
Updating based on acrolinx report

Commit:
edit pass: tutorial-classifying-iris-part-3

Commit:
edit pass: tutorial-classifying-iris-part-2

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into tutorial-classifying-iris-part-2

Commit:
edit pass: tutorial-classifying-iris-part-2

hirokib committed:
added schema and object

paulathurman committed:
edit pass: websocketprotocol

paulathurman committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into websocketprotocol

paulathurman committed:
edit pass: websocketprotocol

Commit:
Doc changes started for mac

Commit:
Updating documentation for docker on Mac

Commit:
some indentation fix

Commit:
Spelling updates, indentations, missing instructions

Commit:
some line-ending fix seen from preview-link

dbreshears committed:
Addressing issues

dbreshears committed:
Attempt Fix of ![NOTE] extension

dbreshears committed:
Fixing [!NOTE] status

Commit:
WIP - quickstart

Commit:
add reference node to TOC

Commit:
minor edit

Commit:
how-to updates

Commit:
remove section that should be in reference

mimig1 committed:
Cosmos DB: Proposed TOC change to contain ATS content.

mimig1 committed:
fixed broken link

v-geberr committed:
edits

mimig1 committed:
acrolinx fixes

mimig1 committed:
update

mimig1 committed:
update

paulathurman committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into websocketprotocol

v-geberr committed:
image fixes, LUIS pub changes

jluk committed:
add portal instructions for launch

paulathurman committed:
edit pass: websocketprotocol

paulathurman committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into websocketprotocol

v-geberr committed:
image size to guidance - as a test for stage server

LoriMerrick committed:
Merge branch 'master' into tutorial-classifying-iris-part-3

paulathurman committed:
edit pass: websocketprotocol

v-geberr committed:
messing with images

paulathurman committed:
edit pass: websocketprotocol

Commit:
edit pass: tutorial-classifying-iris-part-2

Commit:
fixing merge conflicts

v-geberr committed:
image redo according to guidance

rboucher committed:
add MVC marking tag.

genlin committed:
review files

AjayKumar-N committed:
(AzureForums) Added a note : The following instructions are applicable only for unmanaged disks which are the only kind supported

Added a note: "The following instructions are applicable only for unmanaged disks which are the only kind supported" as recommended by anuragdalmia.

mimig1 committed:
update

mimig1 committed:
updates

mimig1 committed:
acrolinx fixes

cephalin committed:
add PS instructions

mimig1 committed:
updates

cephalin committed:
review

YuriDio committed:
retiring article as per PM request

YuriDio committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr

YuriDio committed:
retiring article as per PM request

cephalin committed:
review

Commit:
global-peering-updates

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into global-peering

v-geberr committed:
reverted commit

v-geberr committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into 26647

v-geberr committed:
removed .x

CawaMS committed:
Storage Explorer 0.9.2 release note

v-geberr committed:
Merge branch '26647' of https://github.com/v-geberr/azure-docs-pr into 26647

v-geberr committed:
touched .md

Commit:
change section

YuriDio committed:
fixing error

Commit:
Fixed formatting.

Commit:
Update create-peering-different-deployment-models-subscriptions.md

Commit:
Update create-peering-different-deployment-models.md

Commit:
Update create-peering-different-subscriptions.md

Commit:
whatsnew09

Commit:
Update virtual-network-create-peering.md

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into tutorial-classifying-iris-part-2

v-geberr committed:
altered image, added blue buttons

Commit:
edit pass: tutorial-classifying-iris-part-2

paulathurman committed:
edit pass: cognitive-services-batch1b

Commit:
update UX images, move troubleshooting up toc, reformat tb doc format

Commit:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr

Commit:
add launch icon ref, remove unused imgs

Commit:
add portal icon png

Commit:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr

Commit:
fix merge conflict

Commit:
couple formatting issues

v-geberr committed:
publish at bottom, next button

paulathurman committed:
Update websocketprotocol.md

mimig1 committed:
updates

YuriDio committed:
fixing redirect

Commit:
cortana reference initial commit

Commit:
fix toc

YuriDio committed:
fixing redirect 2

Commit:
edit pass: tutorial-classifying-iris-part-2

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into tutorial-classifying-iris-part-3

Commit:
edit pass: tutorial-classifying-iris-part-3

Commit:
syncing changes

JasonWHowell committed:
Fresh Update batch 2

Commit:
Move table

Commit:
tip

Commit:
language fixes

Commit:
fix duplicate content

Commit:
edits

Commit:
add "use"

Commit:
edits

Commit:
edit pass: tutorial-classifying-iris-part-3

Commit:
headings

CawaMS committed:
Update vs-azure-tools-storage-explorer-relnotes.md

Commit:
remove quotes

ShawnJackson committed:
Update Get-Started-ObjectiveC-iOS.md

hirokib committed:
added comma

JasonWHowell committed:
Fresh Update batch 2

ShawnJackson committed:
Update GetStartedCSharpServiceLibrary.md

ShawnJackson committed:
Update GetStartedClientLibraries.md

hirokib committed:
changed description

CawaMS committed:
Update vs-azure-tools-storage-explorer-relnotes.md

Commit:
section title

GitHubber17 committed:
Merge pull request #26891 from jimdial/global-peering

Global peering registration updates

ShawnJackson committed:
Update tutorial-classifying-iris-part-2.md

Lisaco88 committed:
Merge pull request #25463 from dbreshears/master

Update the Docker Tools TroubleShooting Doc

Commit:
Merge pull request #26933 from JasonWHowell/FreshnessBatch2

PostgreSQL: Fresh Update batch 2

gsacavdm committed:
November update az-gov images

PRMerger12 committed:
Merge pull request #25839 from hirokib/master

added schema and object to EQ tutorial

ShawnJackson committed:
Merge pull request #26824 from paulathurman/websocketprotocol

edit pass: Websocketprotocol

ShawnJackson committed:
Merge pull request #26921 from paulathurman/cognitive-services-batch1b

edit pass: cognitive-services-batch1b

JasonWHowell committed:
Fresh Update batch 3

PRMerger10 committed:
Merge pull request #26924 from Blackmist/freshness

couple formatting issues

ShawnJackson committed:
Merge pull request #25778 from LoriMerrick/tutorial-classifying-iris-part-2

edit pass: tutorial-classifying-iris-part-2

PRMerger19 committed:
Merge pull request #26948 from gsacavdm/patch-2

November update az-gov images

ShawnJackson committed:
Update tutorial-classifying-iris-part-3.md

PRMerger16 committed:
Merge pull request #26951 from JasonWHowell/FreshnessBatch3

Fresh Update batch 3

JasonWHowell committed:
Update concepts-service-tiers.md

ShawnJackson committed:
Merge pull request #25777 from LoriMerrick/tutorial-classifying-iris-part-3

edit pass: tutorial-classifying-iris-part-3

JasonWHowell committed:
Fresh Update batch 4

Commit:
Merge pull request #26953 from JasonWHowell/FreshnessBatch4

Fresh Update batch 4

PRMerger20 committed:
Merge pull request #26952 from JasonWHowell/patch-1

Update concepts-service-tiers.md

Commit:
task 394: escape asterisks, edit procedures to match Verizon GUI

JasonWHowell committed:
Fresh Update batch 5

Commit:
Merge pull request #26957 from JasonWHowell/FreshnessBatch5

Fresh Update batch 5

JasonWHowell committed:
Update concepts-service-tiers.md

PRMerger14 committed:
Merge pull request #21003 from AjayKumar-N/patch-2

(ForumsTeam) A note to highlight if the procedure applies to Managed …

PRMerger17 committed:
Merge pull request #26958 from JasonWHowell/patch-1

Update concepts-service-tiers.md

Commit:
task 394: edit fixes

PRMerger18 committed:
Merge pull request #26911 from MarkusVi/whatsnew09

whatsnew09

Commit:
task 394: edit fixes

tynevi committed:
Merge pull request #26888 from YuriDio/OctoberUpdates_22

October updates 22

Commit:
task 394: edit fixes

Commit:
storage : node code fencing change

Commit:
Merge pull request #26961 from georgewallace/nodeupdate

storage : node code fencing change

Commit:
Merge pull request #26849 from rboucher/patch-9

add MVC marking tag.

Commit:
quickfix

PRMerger15 committed:
Merge pull request #26963 from MarkusVi/quickfix

quickfix

gsacavdm committed:
Fixing syntax highlighting for csharp sections

gsacavdm committed:
Minor updates to gov-functions

PRMerger10 committed:
Merge pull request #26965 from gsacavdm/patch-3

Minor updates to gov-functions

PRMerger17 committed:
Merge pull request #26964 from gsacavdm/patch-2

Fixing syntax highlighting for csharp sections

Commit:
Fix typo in date

Commit:
Fixed bugs.

rayne-wiselman committed:
changing ownership for redirects

rayne-wiselman committed:
Update site-recovery-vmware-to-vmware.md

rayne-wiselman committed:
Update site-recovery-vmware-to-vmware.md

PRMerger18 committed:
Merge pull request #26968 from rayne-wiselman/rayne-november5

changing ownership for redirects

v-thepet committed:
fixing the link

PRMerger17 committed:
Merge pull request #26970 from v-thepet/newfix

production bug: fixing the link

GitHubber17 committed:
Merge pull request #26678 from v-geberr/26647

26647

PRMerger18 committed:
Merge pull request #26852 from genlin/master99

review files

linda33wj committed:
Add more tips for File Storage connector

PRMerger13 committed:
Merge pull request #26972 from linda33wj/master

Add more tips for File Storage connector

linda33wj committed:
Update sub title

Commit:
Merge pull request #26973 from linda33wj/master

Update sub title

linda33wj committed:
Update sub title

Commit:
Merge pull request #26896 from CawaMS/master

Storage Explorer 0.9.2 release note

PRMerger20 committed:
Merge pull request #26974 from linda33wj/master

Update sub title

linda33wj committed:
Update sub title

Commit:
Merge pull request #26976 from linda33wj/master

Update sub title

shengcMSFT committed:
Update concepts-integration-runtime.md

update Location used for Azure Integration Runtime

Commit:
Merge pull request #26977 from shengcMSFT/patch-11

Update concepts-integration-runtime.md

Commit:
addressed PR suggesstions and score improvement tips

Commit:
score improvement tips

dominicbetts committed:
Fix dashes in command

linda33wj committed:
Update sub title

linda33wj committed:
Tweak Teradata version support

Sogup committed:
Change in dates

linda33wj committed:
Revert version update

linda33wj committed:
Fix link

PRMerger17 committed:
Merge pull request #26981 from Sogup/patch-14

Change in dates

Commit:
Merge pull request #26868 from cephalin/nodejs

add PS instructions

PRMerger17 committed:
Merge pull request #26979 from dominicbetts/suite-dash

Fix dashes in command

dominicbetts committed:
Add deploy screenshots

Commit:
Fix.

Commit:
custom-search: add get key step to quickstart

billmath committed:
fixing toc

Commit:
freshness

MikeDodaro committed:
Modify tutorial so user isn't prompted for key until first query.

Commit:
Merge pull request #26980 from linda33wj/master

Update ADF content

rjagiewich committed:
Merge pull request #26923 from jluk/master

update images, reformat to make troubleshooting more findable

rjagiewich committed:
Merge pull request #26955 from dksimpson/dean-t394

task 394: escape asterisks, edit procedures to match Verizon GUI

PRMerger20 committed:
Merge pull request #26988 from Blackmist/freshness

freshness

rjagiewich committed:
Merge pull request #26967 from v-jaswel/master

Fixed bugs.

MikeDodaro committed:
Reformate text <form> tag

PRMerger20 committed:
Merge pull request #26984 from jimdial/patch-25

Fix.

rjagiewich committed:
Merge pull request #26649 from DeniseMak/10-31-ref2

prebuilt entity reference

PRMerger10 committed:
Merge pull request #26990 from MikeDodaro/noPromptWebSrch

Modify tutorial so user isn't prompted for key until first query.

rwike77 committed:
Modify next steps

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

Lisaco88 committed:
Merge pull request #26565 from sayantancs/docker_mac

Service Fabric get started Mac documentation changes for migration to docker

ktoliver committed:
Merge pull request #26669 from mimig1/tabletoc

Cosmos DB: Proposed TOC change to contain ATS content.

Lisaco88 committed:
Merge pull request #26982 from dominicbetts/suite-screenshots

Add deploy screenshots

Lisaco88 committed:
Merge pull request #26986 from brapel/custom-search-qs-get-key

custom-search: add get key step to quickstart

v-alje committed:
Merge pull request #26995 from MicrosoftDocs/FromPublicRepo

From public repo

PRMerger18 committed:
Merge pull request #26993 from rwike77/patch-19

Modify next steps

Lisaco88 committed:
Merge pull request #26987 from billmath/tocfix

fixing toc
